code401challenges/src/test/java/code401challenges/LibraryTest.java

```diff
@@ -16,6 +16,40 @@
         int[]testArray = new int[]{1,2,3,4,5,6};
         int[]result = new int[]{1,2,3,99,4,5,6};
         assertArrayEquals("New Value is inserted into the array", arrayShift.insertShiftArray(testArray, 99),result);
+    }
+
+    //Binary Search
+
```

Comment:
your test look good, im sure you'll figure out the failing test. id like for you to start splitting out tests for each class/code challenge into its own file, otherwise this file will start getting really large.